.github/workflows/integration.yml

```diff
@@ -32,23 +32,73 @@ jobs:
           mode: start
           github-token: ${{ secrets.GH_PERSONAL_ACCESS_TOKEN }}
           ec2-image-id: ami-00b46fa1102c70ff2
-          ec2-instance-type: t2.2xlarge
+          ec2-instance-type: m4.4xlarge
           subnet-id: subnet-905870ae
           security-group-id: sg-0855631d714870b32
 
-  integration:
+  start-executioner:
+    name: Start self-hosted EC2 executioner
+    needs: build
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 6
+      fail-fast: false
+      matrix:
+        include:
+        - id: 1
+          test-grep-filter: 'tag:rpc'
+        - id: 2
+          test-grep-filter: 'tag:boba'
+        - id: 3
+          test-grep-filter: 'tag:mrf'
+        - id: 4
+          test-grep-filter: 'tag:other'
+        - id: 5
+          test-grep-filter: '^((?!tag:).)*$'
```

Comment:
in case there are tests that are not tagged, they somehow slip the PR process - this would catch them